src/shared/icons.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Icon = ({ iconList }) => {
```

Comment:
Komponent Icon jest tak jak nazwa mówi pojedynczym komponentem.
NIe powinien przyjmować .map tylko pojedynczą ikonę.
Jeśli chcesz mieć mapę to robisz to w oddzielnym komponencie, np. NavbarRightIcons.
NabBarIcons nie jest elementem reużywlnym w folderze shared, tylko Icon będzie.

Reply:
zrobione